hathor/cli/run_node.py

```diff
@@ -86,6 +87,16 @@ def prepare(self, args: Namespace) -> None:
         else:
             sys.setrecursionlimit(5000)
 
+        try:
+            import resource
+        except ModuleNotFoundError:
+            pass
+        else:
+            (nofile_soft, _) = resource.getrlimit(resource.RLIMIT_NOFILE)
+            if nofile_soft < 256:
+                print('Maximum number of open file descriptions is too low. Minimum required is 256.')
```

Comment:
Is it 'file descriptions' or 'file descriptors'? Or both are the same?

Reply:
I think this is a typo, should be "file descriptors"

Reply:
Fixed. Thanks!